Ollivanders/src/net/pottercraft/Ollivanders2/Potion/IngredientType.java

```diff
@@ -7,35 +7,51 @@
 
 public enum IngredientType
 {
+   ACONITE (Material.SPIDER_EYE, (short)0, "Aconite"),
    ARMADILLO_BILE (Material.POTION, (short)9, "Armadillo Bile"),
+   ASPHODEL (Material.DEAD_BUSH, (short)0, "Asphodel"),
    BEZOAR (Material.COAL, (short)1, "Bezoar"), // charcoal
    BILLYWIG_STING_SLIME (Material.SLIME_BALL, (short)0, "Billywig Sting Slime"),
    BOOM_BERRY_JUICE (Material.POTION, (short)11, "Boom Berry Juice"),
+   BOOMSLANG_SKIN (Material.ROTTEN_FLESH, (short)0, "Boomslang Skin"),
    BONE (Material.BONE, (short)0, "Bone"),
    BLOOD (Material.POTION, (short)7, "Blood"),
    CHIZPURFLE_FANGS (Material.PUMPKIN_SEEDS, (short)0, "Chizpurfle Fangs"),
    DEATHS_HEAD_MOTH_CHRYSALIS (Material.COAL, (short)0, "Death's Head Moth Chrysalis"),
    DEW_DROP (Material.GHAST_TEAR, (short)0, "Dew Drop"),
    DITTANY (Material.SAPLING, (short)2, "Dittany"), //birch sapling
+   DRAGON_BLOOD (Material.SPLASH_POTION, (short)1, "Dragon Blood"),
+   DRAGONFLY_THORAXES (Material.BEETROOT_SEEDS, (short)0, "Dragonfly Thoraxes"),
    DRIED_NETTLES (Material.SAPLING, (short)0, "Dried Nettles"), // oak sapling
+   FAIRY_WING (Material.GOLD_NUGGET, (short)0, "Fairy Wing"),
+   FIRE_SEEDS (Material.NETHER_WARTS, (short)0, "Fire Seeds"),
    FLOBBERWORM_MUCUS (Material.SLIME_BALL, (short)0, "Flobberworm Mucus"),
+   FLUXWEED (Material.DEAD_BUSH, (short)0, "Fluxweed"),
```

Comment:
Pumpkin or Melon vine might look closer to the picture on http://harrypotter.wikia.com/wiki/Fluxweed

Reply:
I agree, changed it to (Material.VINE)

Reply:
Changed in my push as well